buildSrc/src/main/kotlin/toolkit-intellij-subplugin.gradle.kts

```diff
@@ -131,7 +131,7 @@ afterEvaluate {
 
         systemProperty("aws.telemetry.skip_prompt", "true")
         systemProperty("aws.suppress_deprecation_prompt", true)
-        ciOnly {
+        ciOnly() {
```

Comment:
why this change?

Reply:
IDE suggested it and I had it open, and I was thinking there was an issue with it (there wasn't), also too small to warrant its own PR IMO